src/test/unit/AllocationManagerUnit.t.sol

```diff
@@ -1174,12 +1244,12 @@ contract AllocationManagerUnitTests_SlashOperator is AllocationManagerUnitTests
     }
 
     /**
-     * Allocates all magnitude to a single opSet. Deallocateas magnitude. Slashes al
+     * Allocates all magnitude to a single opSet. Deallocates magnitude. Slashes all
      * Asserts that:
      * 1. The Allocation is 0 after slash
-     * 2. Them sotrage post slash for encumbered and maxMags ais zero
+     * 2. Them storage post slash for encumbered and maxMags is zero
      */
-    function test_allocateAll_deallocateAll() public {
+    function test_slash_allocateAll_deallocateAll() public {
```

Comment:
Would be good to complete the deallocation here and still see that the `allocatable` amount is 0

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/911/commits/26112fe52d6a327ae7da00c4caad2ae227b64e16